internal/datacoord/segment_allocation_policy.go

```diff
@@ -60,27 +62,25 @@ type sealPolicy interface {
 }
 
 // segmentSealPolicy seal policy applies to segment
-type segmentSealPolicy func(*segmentStatus, Timestamp) bool
+type segmentSealPolicy func(status *segmentStatus, info *datapb.SegmentInfo, ts Timestamp) bool
```

Comment:
The policy interface is a little ugly LOL. Maybe we should discuss about it later :(